plugins/net_plugin/net_plugin.cpp

```diff
@@ -3919,6 +4013,16 @@ namespace eosio {
       if( my->find_connection( host ) )
          return "already connected";
 
+      /** @todo when making a shared tls connection, need to use the ctor that takes
```

Comment:
Since setting the participant name has to be handled after the certificate handshake, just remove these comments

Reply:
Removed comment